lang/en/tool_dynamic_cohorts.php

```diff
@@ -69,7 +69,10 @@
 $string['condition:user_enrolment_description'] = 'Users who are {$a->operator} into course "{$a->coursename}" (id {$a->courseid}) with "{$a->role}" role using "{$a->enrolmethod}" enrolment method';
 $string['condition:user_last_login'] = 'User last login';
 $string['condition:user_created'] = 'User created time';
-$string['condition:user_profile']  = 'User standard profile field';
+$string['condition:user_profile'] = 'User standard profile field';
+$string['condition:user_profile_interests'] = 'User interests';
+$string['condition:user_profile_interests_description'] = 'Users interested in any of the following tags {$a}';
```

Comment:
For me this sounds a bit odd. 
How about something like "Users with interests contain following tags {$a}" to be  consistent with other profile data conditions?